docs/tutorials/gpu_executor.md

```diff
@@ -0,0 +1,327 @@
+# Executor on GPU
+
+This tutorial will show you how to use an Executor on a GPU, both locally and in a
+Docker container. You will also learn how to use GPU with pre-built Hub executors.
+
+Using a GPU allows you to significantly speed up encoding for most deep learning models,
+reducing response latency by anything from 5 to 100 times, depending on the model used and 
+inputs. 
+
+Jina enables you to use GPUs like you normally would in a Python script, or in a Docker 
+container - it does not impose any additional requirements or configuration.
+
+```{admonition} Important
+:class: important
+
+This tutorial assumes you are already familiar with basic Jina concepts, such as Document, Executor, and Flow. Some knowledge of the [Hub](../advanced/hub/index) is also needed for the last part of the tutorial.
+
+If you're not yet familiar with these concepts, first read the [Basic Concepts](../fundamentals/concepts) and related documentation, and return to this tutorial once you feel comfortable performing baisc operations in Jina.
+```
+
+## Prerequisites
+
+For this tutorial, you will need to work on a machine with an NVIDIA graphics card. You
+can use various free cloud platforms (like Google Colab or Kaggle kernels), if you do
```

Comment:
can the users easily try this tutorial on google colab or kaggle kernels ?
if it's difficult/not practical to setup docker and jina there, maybe we should not mention them

Reply:
Local version yes, very easy. You can execute shell commands in the notebook using like this
```
! pip install jina
```

for Docker, it is not preinstalled.

Reply:
And googling it, seems it isn't possible to use Docker in colab at all. But anyway, I would leave it up - the first part already covers a lot of ground and might be important for them